app/src/main/java/com/bucketlist/ui/main/MainViewModel.kt

```diff
@@ -9,9 +9,9 @@ import androidx.lifecycle.viewModelScope
 import com.bucketlist.dto.BucketListItem
 
 class MainViewModel : ViewModel() {
-    private var _bucketlistitems: MutableLiveData<ArrayList<BucketListItem>> = MutableLiveData<ArrayList<BucketListItem>>()
+    private var _bucketListItems: MutableLiveData<ArrayList<BucketListItem>> = MutableLiveData<ArrayList<BucketListItem>>()
 
     internal var bucketlistitems:MutableLiveData<ArrayList<BucketListItem>>
-        get () {return _bucketlistitems}
-        set (value) {_bucketlistitems = value}
+        get () {return _bucketListItems}
+        set (value) {_bucketListItems = value}
```

Comment:
changing naming convention to use camel case as recommended here: https://kotlinlang.org/docs/coding-conventions.html#function-names